codebuild/bin/s2n_codebuild.sh

```diff
@@ -106,8 +106,8 @@ run_integration_v2_tests() {
     cd ./build/
     for test_name in $TOX_TEST_NAME; do
       test="${test_name//test_/}"
-      echo "Running... ctest --no-tests=error --output-on-failure --verbose -R ^integrationv2_${test}$"
-      ctest --no-tests=error --output-on-failure --verbose -R ^integrationv2_${test}$
+      echo "Running... ctest --no-tests=error --verbose -R ^integrationv2_${test}$"
+      ctest --no-tests=error --verbose -R ^integrationv2_${test}$
```

Comment:
My CMake build change wouldn't actually affect this call to ctest, since I only modified the environment for unit tests. But this call already has "--verbose", which outputs on failure AND on success, so "--output-on-failure" is unnecessary anyway.